src/Analyzers/CodeAnalysisResult.cs

```diff
@@ -2,24 +2,23 @@
 
 using System.Collections.Generic;
 
-using NonBlocking;
-
 namespace Microsoft.CodeAnalysis.Tools.Analyzers
 {
     internal class CodeAnalysisResult
     {
-        private readonly ConcurrentDictionary<Project, List<Diagnostic>> _dictionary
-            = new ConcurrentDictionary<Project, List<Diagnostic>>();
+        private readonly Dictionary<Project, List<Diagnostic>> _dictionary
```

Comment:
This is fine because there is no concurrency in how we compute diagnostics (that I can see). But my question is why not compute diagnostics in parallel?

Reply:
We run analyzers in parallel.